lib/web/apiserver_test.go

```diff
@@ -6163,6 +6163,72 @@ func TestListConnectionsDiagnostic(t *testing.T) {
 	require.Equal(t, "some details", receivedConnectionDiagnostic.Traces[0].Details)
 }
 
+func TestWebLauncherURL(t *testing.T) {
+	t.Parallel()
+	env := newWebPack(t, 1)
+	proxy := env.proxies[0]
+	pack := proxy.authPack(t, "foo@example.com", nil /* roles */)
+
+	testcases := []struct {
+		desc           string
+		appURL         string
+		wantBadRequest bool
```

Comment:
```suggestion
		code int
```

Then get rid of the if statement below: `require.Equal(t, tt.code, resp.StatusCode)`